File: 02_Tuesday/prac.py
def prac_loops_two():
    factorNum = int(input("Pick a number: "))
    for n in range(1, factorNum):
        if factorNum % n == 0:
            print(f"{n} is a factor of {factorNum}")
        else:
            print(f"{n} is not a factor of {factorNum}")


def prac_diff_loops_two():
    factNum = int(input("Pick a number: "))

    n = 1
    while (n < factNum and n != 0):
        if factNum % n == 0:
            print(f"{n} is a factor of {factNum}")
        else:
            print(f"{n} is not a factor of {factNum}")
        n += 1


def main():
    prac_loops_two()
    prac_diff_loops_two()


if __name__ == "__main__":
    main()

File: 02_Tuesday/prac.js
function pracLoopsTwo() {
  var baseNum = 12;
  var factNum = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11];

  factNum.forEach(function(element) {
    if (element != 0) {
      if (baseNum % element == 0) {
        console.log(`${element} is a factor of ${baseNum}`);
      } else {
        console.log(`${element} is not a factor of ${baseNum}`);
      }
    }
  });
}

function pracDiffLoopsTwo() {
  num = 12;
  baseNum = 1;

  while (baseNum < num) {
    if (baseNum != 0) {
      if (num % baseNum == 0) {
        console.log(`${baseNum} is a factor of ${num}`);
      } else {
        console.log(`${baseNum} is not a factor of ${num}`);
      }
    }
    baseNum += 1;
  }
}

pracLoopsTwo();
pracDiffLoopsTwo();
